doc/testnet.md

```diff
@@ -0,0 +1,25 @@
+# TestNet debugging info
+
+Currently we have two testnets, 'perf' and 'master', both on the master branch of the solana repo. Deploys happen at the top of every hour with the latest code. 'perf' has more cores for the client machine to flood
+the network with transactions until failure.
+
+## Deploy process
+
+They are deployed with the `ci/testnet-deploy.sh` script. Validators are selected based on their machine name and everyone gets the binaries installing from snap.
```

Comment:
what machine/identity runs this thing?

Reply:
They get run through buildkite, so I think it could be multiple agents that pick up the job. I'll add a sentence about that.